telegram/query/dialogs/query.go

```diff
@@ -1,3 +1,3 @@
 package dialogs
 
-//go:generate go run github.com/gotd/td/telegram/query/internal/gen -result=MessagesDialogsClass -package=dialogs -out=queries.gen.go
+//go:generate go run -mod=readonly github.com/gotd/td/telegram/query/internal/gen -result=MessagesDialogsClass -package=dialogs -out=queries.gen.go
```

Comment:
I'm not sure that adding `-mode=readonly` makes sense for internal commands, because there are already in package.

What if we just leave those as is?

Reply:
On the second thought, you are right. I’ve always assumed that `go run` *may* update `go.mod` in some cases—turns that’s not entirely true. Looks like it prints an error even without `-mod=readonly` flag.

**Default**
```
internal/gen/make_errors.go:9:2: no required module provides package example.com/missing-module; to add it:
	go get example.com/missing-module
```

**Read-only**
```
internal/gen/make_errors.go:9:2: cannot find module providing package example.com/missing-module: import lookup disabled by -mod=readonly
```

Thanks for pointing that out!

Reply:
I’ve removed `-mod=readonly` flag in the last commits, so marking this is resolved.